### PR TITLE
Add Deployment badge to README.md.

### PR DESCRIPTION
**Pull request by sunithapatel**
Added a badge to the README.md file for deployment. Added some explanation about deployment and a few more changes.

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Women Who Code Front End Study Group Project
 
+[![Build & Deploy React App](https://github.com/frontendstudygroup/frontendstudygroup.github.io/actions/workflows/ci-build.yml/badge.svg)](https://github.com/frontendstudygroup/frontendstudygroup.github.io/actions/workflows/ci-build.yml)
```
Comment (princiya) [inline README.md:3]: 🎉

File: README.md [hunk 2]
```diff
@@ -20,11 +22,14 @@ Participation and maintenance of this project is completely voluntary and we are
...
-- Clone your forked repository to your pc ( git clone "url from clone option.)
-- Assign the issue to yourself
+- Clone your forked repository to your machine (`git clone [url from clone option]`)
+- Request to assign the issue for yourself
 - Comment on the issue about your approach to solving the problem
 - Get feedback from other contributors or project maintainers
 - Submit a pull request when you are ready
+    - Add a title and small description in the pull request about what you did.
+    - Make sure to reference the issue number in the pull request comment with the words _"Fixes #[issue number]"_ or _"Resolves #[issue number]"_ as explained in [GitHub documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
+
```
Comment (princiya) [inline README.md:32]: do you want to link the pdf you shared on slack about `github for beginners`? you can create a blog entry and link the blog post here? :)

**Comment by princiya** [comment]
🙏  can you also add in the `Expectations` section the gist of the message from [this PR comment](https://github.com/frontendstudygroup/frontendstudygroup.github.io/pull/24#issuecomment-933564326)? i.e. not to use 3rd party code but instead use this study group to learn and build things from scratch.

File: README.md (hunk 2 above)
Comment (sunithapatel) [reply README.md:32]: That pdf file is not available as a link anywhere. I may be able to make it into a blog post and link it here.

**Review by princiya — APPROVED** [review]
thank you @sunithapatel

**Pull request merged**